Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Crew/requisitions.yml

```diff
@@ -401,12 +409,12 @@
     - name: Magazine Boxes
       hasBoxes: true
       entries:
-#      - name: Magazine Box (M10 x 22)
-#        box: RMCMagazine10
-#      - name: Magazine Box (Ext M10 x 14)
-#        box: RMCMagazineExtended10
-#      - name: Magazine Box (Drum M10 x 12)
-#        box: RMCMagazineDrum10
+      - name: Magazine Box (M11 x 22)
+        box: RMCMagazine11
+      - name: Magazine Box (Ext M11 x 14)
+        box: RMCMagazineExtended11
+      - name: Magazine Box (Drum M11 x 12)
+        box: RMCMagazineDrum11
```

Comment:
These three are also broken, they are missing `- id:` at the top which is the id of the box to spawn